commands/store_deploy.go

```diff
@@ -54,6 +56,12 @@ func runStoreDeploy(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("please provide the function name")
 	}
 
+	dockerConfig := configFile{}
+	err := readDockerConfig(&dockerConfig)
```

Comment:
I don't believe there's a need for the users' credentials for any of the images in the store. Those images shouldn't require a login to pull.